elixir/apps/web/lib/web/live/sign_in.ex

```diff
@@ -200,31 +200,46 @@ defmodule Web.SignIn do
         placeholder="Enter your email"
         required
       />
-
-      <:actions>
-        <.button phx-disable-with="Sending..." class="w-full">
-          Request sign in token
-        </.button>
-      </:actions>
+      <.button phx-disable-with="Sending..." class="w-full" style="info">
+        Request sign in token
+      </.button>
     </.simple_form>
     """
   end
 
   def openid_connect_button(assigns) do
     ~H"""
-    <a href={~p"/#{@account}/sign_in/providers/#{@provider}/redirect?#{@params}"} class={~w[
-          w-full inline-flex items-center justify-center py-2.5 px-5
-          bg-white rounded
-          text-sm text-neutral-900
-          border border-neutral-200
-          hover:bg-neutral-100 hover:text-neutral-900
-    ]}>
-      Sign in with <%= @provider.name %>
-    </a>
+    <.button
+      navigate={~p"/#{@account}/sign_in/providers/#{@provider}/redirect?#{@params}"}
+      class="w-full"
+      style="info"
+    >
+      <%= provider_icon(@provider) %> Sign in with&nbsp;<strong><%= @provider.name %></strong>
```

Comment:
I think `&nbsp;` is bad practice, instead you just want to set the container to no-wrap: https://tailwindcss.com/docs/text-wrap